src/pages/all-about-carbon/the-team.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: The team
+description:
+  Carbon is not just one team. To sustain productive contribution and maintain a
+  clear direction for Carbon as a whole, various groups works together to
+  provide direction, governance, and support.
+---
+
+<PageDescription>
+
+Carbon is not just one team. To sustain productive contribution and maintain a
+clear direction for Carbon as a whole, various groups works together to provide
+direction, governance, and support.
+
+</PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Overview</AnchorLink>
+  <AnchorLink>The Carbon team</AnchorLink>
+  <AnchorLink>Steering committee</AnchorLink>
+  <AnchorLink>Working groups</AnchorLink>
+  <AnchorLink>Board of directors</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+Carbon is funded and built by IBM. That means we build for the company’s
+business needs. Carbon is also an open source community-driven project. It is
+available for anyone to use and everyone is encouraged to contribute.
+
+To sustain productive contribution and maintain a clear direction for Carbon as
+a whole, there are various groups within IBM that provide governance and
+support.
+
+## The Carbon team
+
+Carbon is maintained by a dedicated team of designers, developers, and writers
+based in the Austin, Texas IBM Design Studio.
+
+## Steering committee
+
+The Carbon Design System Steering Committee provides oversight and direction for
+Carbon and its design and development community, and related projects.
+
+### Responsibilities
+
+The key responsibilities of the committee are as follows:
+
+- Improve design strategy, quality, and consistency within the design system.
+- Promote and support usage of and contribution to the Carbon Design System
+  within their organizations.
+- Identify and align components/patterns/content that are multi-product or
+  universal.
+- Facilitate consolidation of components and patterns where multiples exist -
+  i.e. “pick winners”.
+- Assess and vote to approve or deny community proposals (e.g. if a particular
+  PAL might warrant the "Carbon for [x]" title, if individuals or teams can be
+  given _committer_ rights to all or part of the Carbon codebase).
+- Approve the creation of working groups and their scope of committed work and
+  recommend members who can contribute.
+- Assist with standardizing information and tooling to support sharing across
+  teams and business units.
+- Provide a quarterly update to the Board of Directors.
+
+### Members
+
+| Member               | Title                                                       |
+| -------------------- | ----------------------------------------------------------- |
+| Sophia Levens, Chair | Executive Director of Design, IBM Sterling, AI Applications |
```

Comment:
Why wouldn't somebody from our team, e.g. Robin, chair the steering committee? By "chair", we mean schedule, facilitate, etc. the meetings, right? It seems like that should be our responsibility.

Reply:
I think there should be a non-core Carbon voice at that level - otherwise it becomes too much of a "Carbon run" thing. Co-chairs, maybe?

Reply:
I'm just thinking from on operational perspective, would somebody outside of our team want that responsibility to coordinate the steering committee.